index.test.js

```diff
@@ -497,6 +498,7 @@ describe('Deploy to ECS', () => {
             .mockReturnValueOnce('cluster-789')         // cluster
             .mockReturnValueOnce('false')               // wait-for-service-stability
             .mockReturnValueOnce('')                    // wait-for-minutes
+            .mockReturnValueOnce('false')               // wait-for-service-stability
```

Comment:
I thought it is defaulted to be false. Or should we fix the comment?

Reply:
Yes, default value is `false`. I've fixed the comment and test